llama.cpp

```diff
@@ -1055,7 +1055,7 @@ struct llama_mmap {
                         strerror(errno));
             }
         }
-        if (numa) {
+        if (numa > 0) {
```

Comment:
Same here.

Reply:
Nice catch. I've fixed that, as well as an instance of treating the enum like an int in a function